### DISCUSSION TITLE
Netrocks + private ssh key

### DISCUSSION THREAD

**Discussion by pafnuteus** [elfmz/far2l#1347] [Q&A]
Здравствуйте.
Имеется far2l version 2.4.0-beta Darwin x86_64
Никак не могу заставить подцеплять приватный ключ для подключения по sftp
При попытке соединиться плагин спрашивает имя и пароль.
Если в терминале коннекчусь стандартным ssh, все норм.

конфиг соединения выглядит следующим образом

```
[myhostname]
Directory=
Host=myhostname.com
LoginMode=0
Options_sftp="Compression:0 ConnectRetries:2 ConnectTimeout:20 CustomSubsystem: MaxReadBlock:32768 MaxWriteBlock:32768 PrivKeyEnable:1 PrivKeyPath:~/.ssh/id_rsa TcpNoDelay:0 TcpQuickAck:0 UseCustomSubsystem:0 UseOpenSSHConfigs:0 "
Port=22
Protocol=sftp
Username=root

```

**Comment by elfmz** [comment]
Имя пользователя в этом случае надо указывать именно что логин под которым конектится к серверу пытаетесь.
А вот пароль - это пароль для приватного ключа, если ключ закрыт паролем. Если нет - пустая строка

**Comment by elfmz** [comment]
если не работает все равно, то
```
export FAR2L_STD=far2l.log
export NETROCKS_VERBOSE=9
far2l

```...и воспроизведите проблему и посмотрите что на предмет ssh в far2l.log напечатается
